tests/ci/test-run-sim-eth.sh

```diff
@@ -33,58 +33,57 @@ function print_log_on_error {
   cat ${SIM_LOGFILE}
 }
 
+function wait_for_licence {
+  i=0
+  while [ ! -f ${HAVE_LICENCE_FILE} ]; do 
+    ((i=i+1))
+    m=$(($i%6))
+    if [[ "$m" -eq "0" ]]; then
+      echo "Tik tok $i";
```

Comment:
Could we change this to "Waiting for license" or similar?

Reply:
And done. But tik tok had its charm